bumble/rfcomm.py

```diff
@@ -263,7 +266,7 @@ def __init__(
         self.window_size = window_size
 
     @staticmethod
-    def from_bytes(data: bytes):
+    def from_bytes(data: bytes) -> 'RFCOMM_MCC_PN':
```

Comment:
this should use the class object rather than the name as a string.

Reply:
(Also in other packet classes) This seems like a Pylance issue(I am using VS Code). Methods referencing its own class cannot be resolved, but mypy considers it's fine.

Reply:
Oh I got it on https://stackoverflow.com/questions/33533148